docs/installation.md

```
@@ -0,0 +1,65 @@
+# Installation
+
+### VS Code
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)